.github/pull_request_template.md

```diff
@@ -27,6 +27,12 @@ https://github.com/DataBiosphere/azul/issues/NUMBER
 - [ ] Added announcement to PR description                  <sub>or this PR does not require announcement</sub>
 - [ ] Added checklist items for additional operator tasks   <sub>or this PR does not require additional tasks</sub>
 
+## Author (hotfixes)
+
+- [ ] Added `h` tag to commit title                         <sub>or this PR does not include a temporary hotfix</sub>
+- [ ] Added `H` tag to commit title                         <sub>or this PR does not include a permanent hotfix</sub>
```

Comment:
For PL: Should author revert a hotfix in the same PR that adds a permanent fix to the issue?

Reply:
From PL: "reversion of temp hotfix should proceed the permanent fix for the issue"